terraform/cloudfoundry/dns-records.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_route53_record" "wildcard" {
+  zone_id = "${var.dns_zone_id}"
+  name = "*.${var.env}.${var.dns_zone_name}."
+  type = "CNAME"
+  ttl = "60"
+  records = ["dummy"]
```

Comment:
Because we don't have it available. It's not created. I have tried to keep this story as small as possible and do not recreate entire alpha environment as we may go in completely different way of setting up networking.